src/main/java/zd/zero/waifu/motivator/plugin/actions/MotivateMeAction.java

```diff
@@ -29,25 +26,20 @@ public void actionPerformed( @NotNull AnActionEvent e ) {
             pluginState.isMotivateMeEnabled(),
             pluginState.isMotivateMeSoundEnabled() );
 
-        ApplicationManager.getApplication().executeOnPooledThread( () ->
```

Comment:
The `MotivationFactory` executes all motivation creation on a pooled thread like this was doing.